app/models/temporary_file.rb

```diff
@@ -1,2 +1,5 @@
+# frozen_string_literal: true
+
 class TemporaryFile < ApplicationRecord
+  has_one :processed_file
```

Comment:
I'm not sure if it's still necessary for 1-1 relationships in rails 5, but just to be safe, can we add a `inverse_of` here?

https://guides.rubyonrails.org/association_basics.html#bi-directional-associations